libraries/aws_dynamodb_table.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsDynamodbTable < AwsResourceBase
```

Comment:
AwsDynamoDbTable would read better.

Reply:
Fair point. I left the `db` part as lowercase as this is how it appears in AWS. Happy to update though.